app/app.go

```diff
@@ -27,40 +28,42 @@ type Application struct {
 func New(environment string) (*Application, error) {
 	var err error
 
+	log.Println("Starting application: environment =", environment)
```

Comment:
As said in the previous review, I would not pollute all functions everywhere with this "environment" variable while it is already available for all.